generator/generator_config.textproto

```diff
@@ -412,14 +412,16 @@ service {
 service {
   service_proto_path: "google/cloud/orchestration/airflow/service/v1/environments.proto"
   additional_proto_files: ["google/cloud/orchestration/airflow/service/v1/operations.proto"]
-  product_path: "google/cloud/composer"
+  product_path: "google/cloud/composer/v1"
```

Comment:
Can we put the new stuff in "google/cloud/orchestration" and still forward from "google/cloud/composer"?

Reply:
I think the build files would get very messy.

It would probably be less effort to:
- check whether `composer` is in use
- if not, claim no break
- move the library, ignoring backwards compatibility

Reply:
Ack.